lib/Mojolicious/Plugin/CGI.pm

```diff
@@ -278,15 +282,15 @@ sub _stdout_cb {
     $buf .= $chunk;
     $buf =~ s/^(.*?\x0a\x0d?\x0a\x0d?)//s or return;    # false until all headers has been read from the CGI script
     $headers = $1;
-
-    if ($headers =~ /^HTTP/) {
+    if ($headers =~ /^HTTP(?: (\d\d\d))?/) {
+      $c->res->code($1) if $1;
```

Comment:
Why do you have HTTP header without a version? Can you provide a link to where this is valid?